File: gita80/if/task2/main.dart
import 'dart:io';

void main() {
  print("KABISA");
  stdout.write("Year: ");
  String yearStr = stdin.readLineSync()??'';
  int year = int.parse(yearStr);
  if(year%400==0) {
    print(366);
  } else {
    if(year%4==0 && year%100!=0) {
      print(366);
    } else {
      print(365);
    }
  }
}

File: gita80/if/task2/main.cpp
#include <iostream>
using namespace std;

int main() {
    cout<<"KABISA YILI"<<endl;
    int year;
    cout<<"Yil: ";
    cin>>year;
    if(year%400==0) {
        cout<<366;
    } else {
        if(year%4==0 && year%100!=0) {
            cout<<366;
        } else {
            cout<<365;
        }
    }
    return 0;
}
